src/components/program/ProgramFAQ.jsx

```diff
@@ -0,0 +1,36 @@
+import { useContext } from 'react';
+import { Collapsible } from '@edx/paragon';
+import { faChevronUp, faChevronDown } from '@fortawesome/free-solid-svg-icons';
+import { FontAwesomeIcon } from '@fortawesome/react-fontawesome';
+import { ProgramContext } from './ProgramContextProvider';
+import './styles/index.scss';
```

Comment:
I don't think this statement is required, css classes belonging to this component are imported anyway in `index.scss`